services/navigation/builtin/builtin.go

```diff
@@ -169,7 +169,9 @@ func (svc *builtIn) startWaypoint() error {
 				return
 			}
 
-			currentLoc, _, err := svc.movementSensor.Position(svc.cancelCtx)
+			// TODO: once https://github.com/viamrobotics/rdk/pull/1545 is merged,
```

Comment:
this is already merged, probably should update here.

Reply:
ah great, will do